inji-verify-sdk/README.md

```
@@ -82,23 +98,33 @@ If summariseResults=false, then response should be
         "schemaAndSignatureCheck": { "valid": true, "error": null },
     "expiryCheck": { "valid": true },
     "statusChecks": [
-        { "purpose": "revocation", "valid": true, "error": null },
-        { "purpose": "suspension", "valid": true, "error": null }
+        { "purpose": "revocation", "valid": true, "error": null }
     ],
         "claims": {...}
 }
 ```
+### Response Fields Summary
+| Property                  | Type    | Required | Description                          |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)